pkg/osbuild/containers_storage_source_test.go

```diff
@@ -0,0 +1,36 @@
+package osbuild
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewContainersStorageSource(t *testing.T) {
```

Comment:
I like this idea a lot. It would definitely be very beneficial. I saw you added a bunch of tests in your PR. I'm assuming those were what you were referring to?